flow-typed/platform.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+declare var Intl: Object;
+
+export type FluxStandardAction = {
```

Comment:
okay cool - yeah I'll change this over to `declare` and make sure it still works as expected - that should keep it aligned with the flow docs a little better as well.